common/messages/messages_en.go

```diff
@@ -22,189 +23,189 @@ import (
 
 // messagesEn ...
 var messagesEn = map[string]util.Message{
-	"AuthenticationFailed": util.Message{
+	"AuthenticationFailed": {
 		Code:        AuthenticationFailed,
 		Description: "Failed to authenticate the user.",
 		Type:        util.Unauthenticated,
 		RC:          400,
 		Action:      "Verify that you entered the correct IBM Cloud user name and password. If the error persists, the authentication service might be unavailable. Wait a few minutes and try again. ",
 	},
-	"ErrorRequiredFieldMissing": util.Message{
+	"ErrorRequiredFieldMissing": {
 		Code:        "ErrorRequiredFieldMissing",
 		Description: "[%s] is required to complete the operation.",
 		Type:        util.InvalidRequest,
 		RC:          400,
 		Action:      "Review the error that is returned. Provide the missing information in your request and try again. ",
 	},
-	"FailedToPlaceOrder": util.Message{
+	"FailedToPlaceOrder": {
 		Code:        "FailedToPlaceOrder",
 		Description: "Failed to create volume with the storage provider",
 		Type:        util.ProvisioningFailed,
 		RC:          500,
 		Action:      "Review the error that is returned. If the volume creation service is currently unavailable, try to manually create the volume with the 'ibmcloud is volume-create' command.",
 	},
-	"FailedToDeleteVolume": util.Message{
-		Code:        "FailedToDeleteVolume",
+	"failedToDeleteVolume": {
```

Comment:
two things 
1- is it mandatory to remove `util.Message`?
2- by changing `failedToDeleteVolume` from `FailedToDeleteVolume`, now we have mix up error code in terms of naming convention, so we need to think how to make it uniform.

Reply:
It is mandatory if we are using golangci, removing util.Message has no effect as it messagesEn is map[string]util.Message. So, its no need to always write util.Message initializing.